contracts/TokenHandler.sol

```diff
@@ -138,6 +137,8 @@ contract TokenHandler is ITokenHandler, ITokenManagerType, ReentrancyGuard, Crea
         // For lock/unlock token managers, the ITS contract needs an approval from the token manager to transfer tokens on its behalf
         if (tokenManagerType == uint256(TokenManagerType.LOCK_UNLOCK) || tokenManagerType == uint256(TokenManagerType.LOCK_UNLOCK_FEE)) {
             ITokenManager(tokenManager).approveService();
+        } else if (tokenManagerType == uint256(TokenManagerType.NATIVE_INTERCHAIN_TOKEN)) {
+            IMinter(ITokenManager(tokenManager).tokenAddress()).transferMintership(tokenManager);
```

Comment:
add a comment explaining why this is needed (something like interchain token bytecode is fixed, but minter needs to be transferred to token manager)